src/containers/ProfileDetails/components/SSID/index.js

```diff
@@ -0,0 +1,360 @@
+            name="local_access"
+            label={
+              <span>
+                <Tooltip title="When a wireless network is configured with 'No Local Access', users will have internet access only. Any traffic to internal resources (other than DHCP and DNS) will be denied.">
+                  <InfoCircleOutlined />
+                </Tooltip>
+                &nbsp; Local Access
+              </span>
+            }
+          >
+            {bridge ? (
+              <>
+                <Radio.Group defaultValue="access">
+                  <Radio value="access">Allow Local Access</Radio>
+                  <Radio value="noAccess">No Local Access</Radio>
+                </Radio.Group>
+              </>
+            ) : (
+              <span className={styles.NotApplicable}>Not Applicable</span>
+            )}
+          </Item>
+
+          <Item name="portal" label="Captive Portal">
+            <>
+              <Radio.Group defaultValue="notPortal">
+                <Radio value="notPortal" onChange={() => setCaptivePortal(false)}>
+                  Do Not Use
+                </Radio>
+                <Radio value="usePortal" onChange={() => setCaptivePortal(true)}>
+                  Use
+                </Radio>
+              </Radio.Group>
+            </>
+            {captivePortal && (
+              <Select className={styles.Field} placeholder="Select Captive Portal Profile">
+                <Option value="default">Default</Option>
+              </Select>
+            )}
+          </Item>
+        </Card>
+
+        <Card title="Security and Encryption">
+          <Item label="Mode">
+            <Select className={styles.Field} onChange={value => setMode(value)}>
+              <Option value="open">Open (No ecryption)</Option>
+              <Option value="wpaPersonal">WPA Personal</Option>
+              <Option value="mixedPersonal" selected>
+                WPA & WPA2 Personal (mixed mode)
+              </Option>
+              <Option value="mixedEnterprise">WPA & WPA2 Enterprise (mixed mode)</Option>
+              <Option value="wpa2Personal">WPA2 Personal</Option>
+              <Option value="wpa2Enterprise">WPA2 Enterprise</Option>
+              <Option value="wep">WEP</Option>
+            </Select>
+          </Item>
+
+          {(mode === 'mixedEnterprise' || mode === 'wpa2Enterprise') && (
+            <Item
+              label="RADIUS Service"
+              name="radius"
+              rules={[
+                {
+                  required: true,
+                  message: 'Please select your RADIUS service',
+                },
+              ]}
+            >
+              <Tooltip title="Add new RADIUS service">
+                <Button icon={<PlusOutlined />} on />
+              </Tooltip>
+            </Item>
+          )}
+
+          {mode !== 'open' &&
+            mode !== 'mixedEnterprise' &&
+            mode !== 'wpa2Enterprise' &&
+            mode !== 'wep' && (
```

Comment:
it should be {(mode === 'wpa' || mode === 'mixedPersonal' || mode === 'wpa2Personal') && <Item ... />}